src/providers/MeetingProvider/MeetingManager.ts

```diff
@@ -162,8 +162,7 @@ class MeetingManager implements DeviceChangeObserver {
         LogLevel.INFO
       );
     }
-    const deviceController = new DefaultDeviceController(logger);
-    configuration.enableWebAudio = false;
```

Comment:
In the original code, `enableWebAudio` is `false`, but you enabled `enableWebAudio` in this PR.

Reply:
Nice catch!!